src/main/java/com/solvd/employeeservice/domain/Employee.java

```diff
@@ -0,0 +1,26 @@
+package com.solvd.employeeservice.domain;
+
+import jakarta.persistence.Entity;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Entity
```

Comment:
@vpetrovaa 
if you want to use hibernate with reactive programming, then [hibernate-reactive](https://hibernate.org/reactive/) is your choice